formbricks/formbricks
# docs: Join the FormTribe blog post

## What does this PR do?



Create a blog post on "Join the FormTribe"

_Note: I noticed none of the images are not displaying on the blog post in dev environment, is this expected?_

https://github.com/formbricks/formbricks/assets/31730715/ef1c4ae8-a8b4-4a2f-8d2f-7651545d14b0


## Type of change




## How should this be tested?



- Start your local environment
- Got to formbricks-web
- Go to blog
- You should see the blog post as the first
- Click it to read

## Checklist
